src/DropdownMenu/DropdownMenu.svench

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { View } from "svench";
+  import DropdownMenu from "./DropdownMenu.svelte";
+  import Button from "../Form/Button.svelte";
+
+  let anchor;
+  let dropdown;
+</script>
+
+<View name="Default">
+  <Button bind:this={anchor} secondary on:click={}>Click Me</Button>
```

Comment:
this on:click handler crashes things, should probably be `on:click={console.log}` or something else? Or does this set some state that opens the dropdown?

Reply:
ahh - this should open the dropdown `on:click={dropdown.show}` ... ill sort that out. thanks